guides/common/assembly_managing-custom-file-type-content.adoc

```diff
@@ -1,17 +1,4 @@
-[id="Managing_Custom_File_Type_Content_{context}"]
-= Managing {customfiletypetitle} Content
-
-In {Project}, you might require methods of managing and distributing SSH keys and source code files or larger files such as virtual machine images and ISO files.
-To achieve this, {customproduct}s in {ProjectName} include repositories for {customfiletype}s.
-This provides a generic method to incorporate arbitrary files in a product.
-
-You can upload files to the repository and synchronize files from an upstream {ProjectServer}.
-When you add files to a {customfiletype} repository, you can use the normal {Project} management functions such as adding a specific version to a Content View to provide version control and making the repository of files available on various {SmartProxyServer}s.
-Clients must download the files over HTTP or HTTPS using `curl -O`.
-
-You can create a file type repository in {ProjectServer} only in a {customproduct}, but there is flexibility in how you create the file type repository.
-You can create an independent file type repository in a directory on the system where {Project} is installed, or on a remote HTTP server, and then synchronize the contents of that directory into {Project}.
-This method is useful when you have multiple files to add to a {Project} repository.
+include::modules/con_managing-custom-file-type-content.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`